tools/tool_settings.h

```diff
@@ -39,6 +39,7 @@
 #include "generated/generated_vulkan_decoder.h"
 #include "util/argument_parser.h"
 #include "util/logging.h"
+#include "util/measurement_manager.h"
```

Comment:
I think this needs to be included in the *_main.cpp files instead of here.

Reply:
Done !